src/adapters/MeetingsJSONAdapter.js

```diff
@@ -37,6 +38,11 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
   getMeeting(ID) {
     return Observable.create((observer) => {
       if (this.datasource[ID]) {
+        // Add the correct local file url to the localVideo object
+        if (this.datasource[ID].localVideo && this.datasource[ID].localVideo.src) {
```

Comment:
You can simplify this and follow the "check" pattern we have
```suggestion
        if (localVideo in this.datasource[ID) {
```
There's no need to check for `src` since you will be assigning it (if it doesn't exist, it gets added)

Reply:
we don't need `in`. I'm doing something simpler